tox.ini

```diff
@@ -44,7 +44,7 @@ allowlist_externals =
     rm
     plantuml
 commands =
-    plantuml -Djava.awt.headless=true -o ../source/_static/ -tpng docs/diagrams/*.puml
+    # plantuml -Djava.awt.headless=true -o ../source/_static/ -tpng docs/diagrams/*.puml
```

Comment:
Can you remove this change?
I have done the same before to comment `plantuml` when it's failing but we don't want it to be commented on by default otherwise it will never be used.

Reply:
I see. I'm trying to fix the issue of test-docs failing in the PR that's why that was commented out as a workaround. But it didn't work.